server/test/services/applicant/question/NameQuestionTest.java

```diff
@@ -57,17 +57,23 @@ public void withEmptyApplicantData_optionalQuestion() {
     assertThat(nameQuestion.getFirstNameValue()).isEmpty();
     assertThat(nameQuestion.getMiddleNameValue()).isEmpty();
     assertThat(nameQuestion.getLastNameValue()).isEmpty();
+    assertThat(nameQuestion.getSuffixValue()).isEmpty();
     assertThat(nameQuestion.getValidationErrors()).isEmpty();
   }
 
   @Test
-  @Parameters({"Wendel,Middle Name,Patric", "Wendel,,Patrick"})
+  @Parameters({"Wendel,Middle Name,Patric,Suffix", "Wendel,,Patrick,"})
   public void withValidApplicantData_passesValidation(
-      String firstName, String middleName, String lastName) {
+      String firstName, String middleName, String lastName, String suffix) {
     ApplicantQuestion applicantQuestion =
         new ApplicantQuestion(nameQuestionDefinition, applicantData, Optional.empty());
     QuestionAnswerer.answerNameQuestion(
-        applicantData, applicantQuestion.getContextualizedPath(), firstName, middleName, lastName);
+        applicantData,
+        applicantQuestion.getContextualizedPath(),
+        firstName,
+        middleName,
+        lastName,
+        suffix);
```

Comment:
**issue** I don't see any updates to the asserts to make sure the new suffix field is updated correctly

Reply:
Ah good catch, added validation to make sure suffix is updated correctly.